src/pydata_sphinx_theme/assets/styles/abstracts/_accessibility.scss

```diff
@@ -49,25 +49,25 @@
 // Return WCAG2.1 relative luminance
 // See https://www.w3.org/TR/WCAG/#dfn-relative-luminance
 // See https://www.w3.org/TR/WCAG/#dfn-contrast-ratio
-//
 @function luminance($target-color) {
   $rgb-col: (
-    "r": red($target-color),
-    "g": green($target-color),
-    "b": blue($target-color),
+    "r": color.red($target-color),
+    "g": color.green($target-color),
+    "b": color.blue($target-color),
   );
 
   @each $channel, $value in $rgb-col {
     // here we get RsRGB, GsRGB, and BsRGB
-    @if math.div($value, 255) <=0.03928 {
-      $rgb-col: map-merge(
+    // stylelint-disable-next-line number-max-precision
+    @if math.div($value, 255) <= 0.03928 {
```

Comment:
Note to self: investigate whether I should add a stylelint ignore or if I really should adjust the precision of the number or the right hand side of the comparison

Reply:
Comes from WCAG definition. This Stylelint rule is meant for CSS style rules, not SCSS functions, so the lint failure for this line is definitely a false positive.

However, @trallard, note:

> Before May 2021 the value of 0.04045 in the definition was different (0.03928). It was taken from an older version of the specification and has been updated. It has no practical effect on the calculations in the context of these guidelines.
> &hyphen; https://www.w3.org/TR/WCAG22/#dfn-relative-luminance

Reply:
Created issue so I don't forget: https://github.com/pydata/pydata-sphinx-theme/issues/1835